src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -16,6 +16,11 @@
  * Represent a Parser for OpenCommand.
  */
 public class OpenCommandParser implements CommandParser<OpenCommand> {
+    private static final String ERROR_PATH = "The Path \"%s\" does not exist in the storage. "
+            + "Please provide a valid Path";
+    private static final String ERROR_EMPTY_PATH = "Empty Path detected. "
+            + "Please provide a valid Path";
```

Comment:
Nit: maybe we can just say "Path cannot be empty. Please provide a valid path."? What do you think? And I'm not really sure why you capitalise the word "Path" too (for everything) actually, but it's up to you I think it's no biggie.